src/pages/Settings/EmailSettings.vue

```diff
@@ -445,7 +462,8 @@ export default {
 				networkTransactions: false,
 				networkDigest: false,
 				trusteeNews: false,
-				teamMessageFrequencies: []
+				teamMessageFrequencies: [],
+				borrowerFullyFunded: 'weekly'
```

Comment:
Being that you have to reference the `weekly` value repeatedly, it would be good to create a constant up above your export statement to make that more reuseable... something like  `const BORROWER_FUNDED_EMAIL_DEFAULT = 'weekly';`

Reply:
👍